README.md

```diff
@@ -73,20 +73,25 @@ To install prerequisite packages on the Platform9 Express control host, run the
 ## Configuration Inventory (CLI Only)
 Platform9 Express uses Ansible to execute commands on the hosts to be taken under management.  In order to configure Ansible to run remote commands on the managed hosts, the Ansible Inventory file must be configured.  This file is located in /opt/pf9-express/inventory/hosts.
 
-NOTE: A sample template is installed in the previous command ("./pf9-express -s").
+NOTE: A sample template is installed in the previous command ("./pf9-express -s"). Breaking down the Inventory File to separate Openstack and Kubernetes below:
```

Comment:
I'd modify to "A breakdown of the inventory file is below:"

Reply:
corrected